pages/dkp/kommander/2.2/clusters/attach-cluster/manually-attach-cluster/index.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: layout.pug
+navigationTitle: Manually attach Cluster created with CLI
+title: Attach a Cluster manually created with CLI
+menuWeight: 50
+excerpt: Attach a Cluster manually created with CLI
+---
+
+When creating a cluster in a Workspace namespace using the DKP CLI it will not be attached automatically. To automatically attach a cluster you can generate the cluster objects using the DKP CLI `--dry-run -o yaml` flags and then using the [Advanced Creation of Konvoy Clusters][create_cluster_advanced].
+
+If the Cluster is created using the CLI it will appear in the UI as it is being Provisioned. It will then be in an Unattached state when Provisioning is completed.
```

Comment:
```suggestion
If you created the cluster using the CLI, it will appear in the UI as Provisioned. Once provisioning is completed, the state changes to Unattached.
```

Reply:
@gvasquezvargas the changes you made are not completely accurate.

> it will appear in the UI as Provisioned.

The status will go `Pending` -> `Provisioning` -> `Unattached` in this scenario, the user will never see the `Provisioned` status.

>  Once provisioning is completed, the state changes to Unattached.

I think `status` is preferable to `state` here, IMO.

Reply:
Oh, I see! Thank you for clarifying that @TattdCodeMonkey ! Maybe this one is better? 
```suggestion
When you create a cluster using the CLI, you will be able to see it in the UI while it is being provisioned. Once provisioning is completed, the status will change to Unattached.
```